charts/shield/values.yaml

```
@@ -140,6 +140,11 @@ features:
       enabled: false
       integrations: []
       transmit_message_types: []
+      # If existing_secret_name is set to anything different than an empty string
+      # the local forwarder secret generation is skipped and the existing secret
+      # name is used to mount the secret volume instead of the one defined in
+      # the secrets template
+      existing_secret_name: ""
```

```suggestion
      existing_secret: ""
```
let's align with all the other existing_secrets removing the `_name` suffix
maybe we can add a prefix to clarify what it replaces